src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -2,662 +2,838 @@
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
```

Comment:
you can trim some of the unused xmlnses here, especially the new ones added in this pr